features/RailsVersions.md

```diff
@@ -8,3 +9,5 @@
     4.2           | >= 3.1
     5.0           | >= 3.5
     5.1           | >= 3.6
+    5.2           | >= 3.8
```

Comment:
It might be that Rails 5.2 was supported in 3.6/3.7, but in 3.8 there were some fixes to improve Rails 5.2 support.

Reply:
👍